nextcord/application_command.py

```diff
@@ -722,6 +761,12 @@ def from_callback(
         if self.name is None:
             self.name = self.callback.__name__
 
+        if self.error_callback is None:
+            if self.parent_cog:
```

Comment:
Couldn't you combine these two if statements together with an and statement?

Reply:
Done